src/services/usageTracking/index.ts

```diff
@@ -106,3 +117,13 @@ async function trackInstallation() {
     log.error('Error tracking installation:', error)
   }
 }
+
+async function getLoginState(): Promise<boolean> {
+  try {
+    const { profiles } = await getProfileData()
```

Comment:
Looking for suggestions here: `getProfileData` reads data from disk, which is something I'd rather avoid as I wouldn't want the app to read from disk each time a usage event is send

Reply:
Thank you!